### ISSUE TITLE
The SDK has not been initialized, make sure to call FacebookSdk.sdkInitialize() first. 

### ISSUE THREAD

**Issue by mishrasatyam** [robingenz/capacitor-firebase-authentication#64]
**Current behavior:**
I just installed @robingenz/capacitor-firebase-authentication@0.3.6 . I did not use it in the code and after sync with capacitor I get  "The SDK has not been initialized, make sure to call FacebookSdk.sdkInitialize() first. "

**Expected behavior:**
Above behaviour should not happen . This is happening when I haven't even imported anything from the package yet. 

**Capacitor doctor:**

```
   Capacitor Doctor   

Latest Dependencies:

  @capacitor/cli: 3.2.0
  @capacitor/core: 3.2.0
  @capacitor/android: 3.2.0
  @capacitor/ios: 3.2.0

Installed Dependencies:

  @capacitor/ios: not installed
  @capacitor/core: 3.2.0
  @capacitor/cli: 3.2.0
  @capacitor/android: 3.2.0

[success] Android looking great! 👌
```

**Comment by robingenz** [comment]
Hi @mishrasatyam,
I tried to reproduce the problem, but unfortunately I did not get it.
I do not receive any error message after sync.
Can you please provide me a Minimal, Reproducible Example (see [here](https://stackoverflow.com/help/minimal-reproducible-example)) and the exact steps?

**Comment by pradeepmdk** [comment]
@mishrasatyam add below in MainActivity.java
public class MainActivity extends BridgeActivity {
  @Override
  public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    FacebookSdk.sdkInitialize(getApplicationContext());
  }
}

**Comment by robingenz** [comment]
This should not be necessary.
If you provide a minimal, reproducible example, I will try to fix the problem.

**Comment by mishrasatyam** [comment]
Hey , I have created a [repo](https://github.com/mishrasatyam/capacitor_auth_test) . Please check . Thanks.

**Comment by robingenz** [comment]
@mishrasatyam Please follow the installation instructions correctly (see [here](https://github.com/robingenz/capacitor-firebase-authentication/blob/main/docs/setup-facebook.md)):
1. `android/app/src/main/res/values/strings.xml` does not contain the required configuration values
2. `android/app/src/main/AndroidManifest.xml` does not contain the required configuration values

Also see [Before you begin](https://firebase.google.com/docs/auth/android/facebook-login#before_you_begin) and follow the instructions to configure sign-in with Facebook correctly.  

Please update your example and check if the problem still exists.

**Comment by robingenz** [comment]
Oh sorry, you didn't mean to use Facebook did you? You just installed the plugin. So the setup is correct. 
I tried to reproduce the issue with your repo but get the following error messages:
```bash
$ npx cap sync android
[error] Could not find installation of TypeScript.
        To use capacitor.config.ts files, you must install TypeScript in your project, e.g. w/ npm install -D typescript

$ npm install -D typescript

added 1 package, and audited 209 packages in 2s

13 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

$ npx cap sync android
[error] Could not find the web assets directory: .\build.
        Please create it and make sure it has an index.html file. You can change the path of this directory in
        capacitor.config.ts (webDir option). You may need to compile the web assets for your app (typically npm run
        build). More info: https://capacitorjs.com/docs/basics/workflow#sync-your-project
```
It seems that the project is not yet configured correctly.

I have created my own test project:
```
npx ionic start
npx ionic build
npm install @robingenz/capacitor-firebase-authentication firebase
npx ionic cap add android
```
I get the following output after running `npx cap sync`:
```
√ Copying web assets from www to android\app\src\main\assets\public in 1.67s
√ Creating capacitor.config.json in android\app\src\main\assets in 1.66ms
√ copy android in 1.72s
√ Updating Android plugins in 8.56ms
[info] Found 4 Capacitor plugins for android:
       @capacitor/app@1.0.3
       @capacitor/haptics@1.0.3
       @capacitor/keyboard@1.0.3
       @robingenz/capacitor-firebase-authentication@0.3.9
√ update android in 104.04ms
√ copy web in 1.10ms
√ update web in 979.90μp
[info] Sync finished in 1.83s
```
So everything is fine. I still can't reproduce it.

Do you have any more information for me? What are the exact steps you take?

**Comment by mishrasatyam** [comment]
Hey, just do npm install and npm run android in the repo i linked. It will build and open android studio. You can see error when you try to run on a Device

**Comment by robingenz** [comment]
I was able to reproduce it and will take a closer look the next few days. Thank you!

**Comment by robingenz** [comment]
As a temporary workaround, the Facebook provider should be disabled (if it is not needed) via the [`providers` configuration option](https://github.com/robingenz/capacitor-firebase-authentication#configuration). 
I will publish a fix today.

**Comment by robingenz** [comment]
The message appears because the Facebook SDK does not find the required configurations.
The SDK is required for this plugin to provide Facebook sign-in functionality.
The message now no longer causes the app to crash.
The fix will be included in version `4.0.0`.
